umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -241,6 +242,15 @@
      (dissoc :ComposedOf)
      (dissoc :NumberOfInstruments)))
 
+(defn migrate-tiling-identification-systems
+  "Migrate TilingIdentificationSystems from 1.9 up to 1.10"
+  [collection]
+  (let [tiling-id-systems (filter
+                           #(spatial-conversion/tile-id-system-name-is-valid?
+                             (:TilingIdentificationSystemName %))
+                           (:TilingIdentificationSystems collection))]
+   (assoc collection :TilingIdentificationSystems (seq tiling-id-systems))))
```

Comment:
I agree with @sxu123, we should change the line to:
```
(if (seq tiling-id-systems)
 (assoc collection :TilingIdentificationSystems titling-id-systems)
 collection)
```
The expected result in test should be updated to match what is generated here.